config-linux.md

```diff
@@ -598,6 +598,9 @@ The following parameters can be specified to set up seccomp:
     * `SECCOMP_FILTER_FLAG_TSYNC`
     * `SECCOMP_FILTER_FLAG_LOG`
     * `SECCOMP_FILTER_FLAG_SPEC_ALLOW`
+    * `SECCOMP_FILTER_FLAG_NEW_LISTENER`
+
+* **`listenerPath`** *(string, OPTIONAL)* - specifies the path of UNIX domain socket which the runtime will pass the file descriptor of seccomp notification using SCM_RIGHT to.
```

Comment:
We may want to prioritize the DLL interface rather than socket. https://github.com/containers/crun/pull/438

cc @giuseppe

Reply:
do you think that should be part of the specs?  I considered that more of a development tool.  How would the .so handled?  Through cgo?

Reply:
I'm sorry for the very late reply.
DLL interface is useful but I think we don' t need it as runtime-spec as @giuseppe said.